faseb-journal.csl

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" initialize-with=", " names-delimiter=", " page-range-format="expanded" default-locale="en-US" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>FASEB Journal</title>
+    <title-short>FASEB</title-short>
```

Comment:
I'd rather not use the very long name of the journal as "title", because "title" is what will appear e.g. on zotero.org/styles, but everybody would be searching for "FASEB", which is what this journal is commonly called, and nobody would find it if the full title was spelled out.